httpclient-backend/zio/src/main/scala/sttp/client3/httpclient/zio/ZioServerSentEvents.scala

```diff
@@ -0,0 +1,15 @@
+package sttp.client3.httpclient.zio
+
+import sttp.capabilities.zio.ZioStreams
+import sttp.client3.sse.ServerSentEvent
+import zio.stream.ZTransducer
+
+object ZioServerSentEvents {
```

Comment:
this should go to `implementations/zio` as well, like with `fs2` (thus avoiding duplication)